README.md

```diff
@@ -140,3 +140,22 @@ Where `"name or id"` is a name or id from the `lead_providers` table.
 1. Get into Rails console for the environment you want to generate the token for.
 2. Run `EngageAndLearnApiToken.create_with_random_token!`
 3. Rails console should output a string, that's your unhashed token, you can keep it and use it to access E&L endpoints.
+
+## payment_calculator
+
+The code in [`lib/payment_calculator/`](lib/payment_calculator/) performs payment calculations for both [ECFs (Early Career Framework)](https://www.early-career-framework.education.gov.uk/) and [the reformed NPQs (National Professional Qualification)](https://www.gov.uk/government/publications/national-professional-qualifications-frameworks-from-september-2021) so that training providers can be paid the correct amount.
```

Comment:
I'm reluctant to link to the dreaded wordpress site from the readme, especially as it's going offline soon-ish

Reply:
I think it adds useful context personally, how strongly do you want rid it? If you really want rid of it suggest a change and I'll accept it.

Reply:
Not strongly at all, just a suggestion. We often link out to https://www.gov.uk/government/publications/early-career-framework-reforms-overview/early-career-framework-reforms-overview FWIW

Reply:
I think it's alright to link there, before we delete it we (E&L) will go through all the places we know of that use it and change them, and R&P copy is one of those places.